src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -287,16 +287,26 @@
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxScreensaver"/>
       </item>
-      <item row="5" column="0">
+      <item row="6" column="0">
        <widget class="QLabel" name="multiSamplingLabel">
         <property name="text">
          <string>Multi-Sampling</string>
         </property>
        </widget>
       </item>
-      <item row="5" column="1" colspan="2">
+      <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="multiSamplingComboBox"/>
       </item>
+      <item row="7" column="0" colspan="2">
+       <widget class="QCheckBox" name="checkBoxForceHardwareAcceleration">
+        <property name="text">
+         <string>Force 3D acceleration</string>
+        </property>
+        <property name="toolTip">
+         <string>If checked, Mixxx will always assume 3D acceleration is available. This may lead to pour performance if only CP-backed rendering is available..</string>
```

Comment:
thanks for the improved comment. I think this now needs an explanation why this is necessary. Without further context this just looks like a footgun.

Reply:
Shouldn't qt check this automatically?

Reply:
The usecase for this feature is to conditionally enable graphical effects, such as blurred background. I'm sure there is some Qt predicate we could use eventually, but I haven't solved it yet. This setting just allows me to move forward for now basically.

Reply:
I see, so you disable the effects completely unless this flag is on?

Reply:
Yes exactly, or use a "cpu-friendly" effect (in this case opacity)